port/unix/omrsysinfo.c

```diff
@@ -604,20 +608,77 @@ omrsysinfo_get_OS_version(struct OMRPortLibrary *portLibrary)
 {
 	if (NULL == PPG_si_osVersion) {
 		int rc;
```

Comment:
While you're here, please initialize rc.

Reply:
Will do.